### PR TITLE
some update to this page

### PR DESCRIPTION
**Pull request by linsun**
there is probably more to update

[ ] Configuration Infrastructure
[ ] Docs
[x ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: operator/ARCHITECTURE.md [hunk 1]
```diff
@@ -93,24 +91,26 @@ namespace is defined as:
...
+  namespace: istio-gateways
+spec:
   components:
-    namespace: istio-security
-    citadel:
-policy:
-  components:
-    policy:
+    ingressGateways:
+    - name: istio-ingressgateway
```
Comment (morvencao) [inline operator/ARCHITECTURE.md:104]: Nit: is this example to deploy gateway to `istio-gateways` ns? If yes, we can add namespace here `namespace: istio-gateways`

Comment (linsun) [reply operator/ARCHITECTURE.md:104]: @morvencao yes, it is to show how to deploy to a namespace.   sorry I don't understand your comment... I had the namespace in line 100.

Comment (morvencao) [reply operator/ARCHITECTURE.md:104]: I mean this example will deploy gateway into `istio-system` namespace, but your intention is to deploy gateway to `istio-gateways` namespace, right?

If yes, then I suggest to add line `namespace: istio-gateways` after line 104.

Comment (linsun) [reply operator/ARCHITECTURE.md:104]: sorry I missed your comment earlier!  It is addressed now. :)  PTAL

**Review by morvencao — APPROVED** [review]

**Pull request merged**